Src/Scripting/NativeDefinitions/MathDefinitions.cpp

```diff
@@ -106,7 +108,9 @@ MathDefinitions::MathDefinitions(ScriptManager* mgr) {
     engine->RegisterObjectMethod("Vector2f", "Vector2f opMul(float f) const", asMETHODPR(PGE::Vector2f, operator*, (float) const, PGE::Vector2f), asCALL_THISCALL);
     engine->RegisterObjectMethod("Vector2f", "Vector2f opMul_r(float f) const", asMETHODPR(PGE::Vector2f, operator*, (float) const, PGE::Vector2f), asCALL_THISCALL);
     engine->RegisterObjectMethod("Vector2f", "Vector2f opDiv(float f) const", asMETHODPR(PGE::Vector2f, operator/, (float) const, PGE::Vector2f), asCALL_THISCALL);
-    
+    engine->RegisterObjectMethod("Vector2f", "Vector2f opDiv(const Vector2f&in other) const", asMETHODPR(PGE::Vector2f, operator/, (float) const, PGE::Vector2f), asCALL_THISCALL);
+    engine->RegisterObjectMethod("Vector2f", "Vector2f opMul(const Vector2f&in other) const", asMETHODPR(PGE::Vector2f, operator*, (float) const, PGE::Vector2f), asCALL_THISCALL);
```

Comment:
These overloads are not permissible.

Reply:
See earlier comment, multiply is the most common purpose for *.

Reply:
This has been reverted for the time being.